src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -440,33 +465,37 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
               Buy now
             </Button>
           )}
-          {artwork.is_offerable && (
+          {shouldDisplayMakeOfferButton() && (
             <>
               <Spacer mt={2} />
               <Button
-                variant={
-                  artwork.is_acquireable ? "secondaryOutline" : "primaryBlack"
-                }
+                variant={isAcquireable ? "secondaryOutline" : "primaryBlack"}
                 width="100%"
                 size="medium"
                 loading={isCommittingCreateOfferOrderMutation}
                 onClick={this.handleCreateOfferOrder.bind(this)}
+                data-test-id="makeOfferButton"
               >
                 Make offer
               </Button>
             </>
           )}
-          {artwork.is_inquireable &&
-            !artwork.is_acquireable &&
-            !artwork.is_offerable && (
-              <Button
-                width="100%"
-                size="medium"
-                onClick={this.handleInquiry.bind(this)}
-              >
-                Contact Gallery
-              </Button>
-            )}
+          {shouldDisplayContactGalleryButton() && (
+            <Button
+              width="100%"
+              size="medium"
+              onClick={this.handleInquiry.bind(this)}
+              mt={shouldDisplayMakeOfferAsPrimary() ? 2 : 0}
+              variant={
+                shouldDisplayMakeOfferAsPrimary()
+                  ? "secondaryOutline"
+                  : "primaryBlack"
+              }
+              data-test-id="contactGalleryButton"
```

Comment:
I don't think we need a test-id here to select this button in the tests.
You can use(RTL): 
```javascript
screen.getAllByRoles("button")[desiredIndex]
screen.getByText("Contact Gallery")
```
both cases are better than test ids, test ids is the last resort if we can't query the component in any possible way

Reply:
Noted about RTL but I would push back about grabbing components by their index rather than a specific property on the component, e.g. a test id bc if the UI changes, e.g. a new button, then we have to update the tests to reflect what could be a moving target (new UI elements on web page)

Reply:
Interesting discussion here... I also thought that using test-ids would be preferable, for the same reasons that @ashleyjelks mentioned... I would be curious to learn more about the downsides of this approach, as there's probably something that I'm not seeing...

Reply:
@lidimayra test-ids are only recommended when you have no other choice, that's why it's in the lower level of priority in the [React testing library documentation](https://testing-library.com/docs/queries/about/#priority)

@ashleyjelks I agree about the index, but the getByText sill be useful in this case and has a higher priority over test-id queries

Reply:
@araujobarret Cool, many thanks for the clarification and the reference 🙌